pkg/agent/variables.go

```diff
@@ -178,6 +178,10 @@ func getOutBoundCmd(nbc *datamodel.NodeBootstrappingConfiguration, cloudSpecConf
 		return ""
 	}
 
+	if strings.ToLower(nbc.OutboundType) == datamodel.OutboundTypeBlock || strings.ToLower(nbc.OutboundType) == datamodel.OutboundTypeNone {
```

Comment:
probably better to just use `strings.EqualFold(nbc.OutboundType, datamodel.OutboundTypeBlock)`, etc.